src/sorcha/utilities/sorchaGetLogger.py

```diff
@@ -3,7 +3,7 @@
 from datetime import datetime
 
 
-def PPGetLogger(
+def GetLogger(
```

Comment:
There is a getLogger command with the log file class so GetLogger can be confusing to someone to realize that this our custom one, so best to call this sorchaGetLogger if possible